### ISSUE TITLE
[Anon] Bug report: Imperium - Space Marines.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#3776]
**File:** Imperium - Space Marines.catz

**BattleScribe version:** 2.01.15

**Platform:** Windows

**Dropbox:** Yes

**Description:** Imperium - Space Marines - Chapter Selection = Crimson Fists
 Cannot choose CF-specific relic on eligible characters. Ex: Captain with power fist should be able to choose Fist of Vengeance. This is not appearing in list of available relics.

**Comment by acebaur** [comment]
You have to select relics of the chapter from the no force Org chart for it to show up. However I am looking into why there is no free CP option for the first one.

**Comment by AluminumMan** [comment]
Yes, relics of the chapter is also selected, tried both versions (1CP & 3CP) and it's still not showing. Non-chapter-specific relics do show, just not this one.
